frontend/src/features/projects/assess/components/ReviewApproveActions.test.tsx

```diff
@@ -36,6 +38,7 @@ describe('Review Approve actions', () => {
   describe('approval button', () => {
     (useFormikContext as jest.Mock).mockReturnValue({
```

Comment:
I want to comment on this, because I don't think this works the way the author thought it does.

Anything not included in a before() after() it or similar block runs simultaneously (not only during the test) In addition, mocking a value is a global action. 

This test mocks useFormikContext multiple times. All of those mocks affect all of the tests if you want to add something to the mocked formik object, you need to add id to all instances of this mock.

I have plans to circle back and fix this test, but just sharing what I learned here.